report/renderer/performance-category-renderer.js

```diff
@@ -220,6 +289,17 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
         this.dom.registerSwappableSections(legacyAuditsSection, experimentalInsightsSection);
       }
     }
+    // Deal with the user loading the report and having toggled to Insights
+    // which is now stored in local storage. Put in a rAF otherwise this code
+    // runs before the DOM is created.
+    if (this._getInsightToggleState() === 'INSIGHTS') {
+      requestAnimationFrame(() => {
```

Comment:
this felt messy to me but until this function returns the content isn't actually appended; feedback welcome on alternative ways to do this.